### PR TITLE
feat(balance): update capacity of power armor holsters

### PR DESCRIPTION
**Pull request by chaosvolt**
## Purpose of change

Updates some items I overlooked back when I updated holster sizes in https://github.com/cataclysmbnteam/Cataclysm-BN/pull/3159, plus expand on one item's role.

## Describe the solution

1. Per Scarf's suggestion, increased max volume of back holster from 9 liters to 10 liters, matching that of power armor back holster.
2. Increased maximum volume of power armor holster from 800 ml to 1 liter and minimum volume from 100 ml to 250 ml, matching standard holster.
3. Altered the power armor chest rig a bit, renaming it to "power armor magazine carrier" to distinguish it from from the carry stats of the standard chest rig. Reduced its max mags down to only 2, but bumped its max volume to 1500 ml and allowed it to hold items with the `MAG_BULKY` flag. This gives it the ability to stash spare heavy batteries, like what a UPS loads.

## Describe alternatives you've considered

1. Allowing the mag pouches to also hold ammo belts via adding a flag to it. 5.56 and 7.62 have a pretty massive volume when maxed out at 500 rounds, and even if toned down to 200 that'd require the mag carrier's capacity to be boosted a bit higher, while it'd hold a 200-round 5.56mm belt a 200-round 7.62mm belt is still just a hair over 2.5 liters.
2. Allowing more mag carrier items to hold `MAG_BULKY` items.

## Testing

Checked affected file for syntax and lint errors.

### CODE REVIEW

**Comment by scarf005** [comment]
i think increasing volume of both back holster to 10L might be a bit better.

**Comment by chaosvolt** [comment]
Sure, will do. :>

File: data/json/items/armor/power_armor.json [hunk 1]
```diff
@@ -471,10 +471,11 @@
       "type": "holster",
       "holster_prompt": "Stash ammo",
       "holster_msg": "You stash your %s.",
-      "multi": 3,
-      "max_volume": "750 ml",
-      "draw_cost": 40,
-      "flags": [ "MAG_COMPACT" ]
+      "multi": 4,
```
Comment (scarf005) [inline data/json/items/armor/power_armor.json:474]: description says it can hold 2 magazines, yet `"multi"` is set to 4. is this intentional?

Comment (chaosvolt) [reply data/json/items/armor/power_armor.json:474]: Whoops, copypaste error on my part.

**Review by scarf005 — APPROVED** [review]
I think it'll be OK to allow three mags tho

**Pull request merged**